runelite-client/src/main/java/net/runelite/client/plugins/timetracking/clocks/ClockTabPanel.java

```diff
@@ -0,0 +1,191 @@
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.timetracking.clocks;
+
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import java.awt.event.ActionListener;
+import java.awt.image.BufferedImage;
+import java.util.ArrayList;
+import java.util.List;
+import javax.swing.ImageIcon;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.EmptyBorder;
+import net.runelite.client.plugins.timetracking.TabContentPanel;
+import net.runelite.client.plugins.timetracking.TimeTrackingPlugin;
+import net.runelite.client.ui.ColorScheme;
+import net.runelite.client.ui.DynamicGridLayout;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.components.IconButton;
+import net.runelite.client.ui.components.shadowlabel.JShadowedLabel;
+import net.runelite.client.util.ImageUtil;
+
+public class ClockTabPanel extends TabContentPanel
+{
+	static final ImageIcon DELETE_ICON;
+	static final ImageIcon DELETE_ICON_HOVER;
+	static final ImageIcon LAP_ICON;
+	static final ImageIcon LAP_ICON_HOVER;
+	static final ImageIcon PAUSE_ICON;
+	static final ImageIcon PAUSE_ICON_HOVER;
+	static final ImageIcon RESET_ICON;
+	static final ImageIcon RESET_ICON_HOVER;
+	static final ImageIcon START_ICON;
+	static final ImageIcon START_ICON_HOVER;
+
+	private static final ImageIcon ADD_ICON;
+	private static final ImageIcon ADD_ICON_HOVER;
+
+	private final ClockManager clockManager;
+
+	private final List<ClockPanel> clockPanels = new ArrayList<>();
+
+	static
+	{
+		BufferedImage deleteIcon = ImageUtil.getResourceStreamFromClass(TimeTrackingPlugin.class, "delete_icon.png");
+		BufferedImage lapIcon = ImageUtil.getResourceStreamFromClass(TimeTrackingPlugin.class, "lap_icon.png");
+		BufferedImage pauseIcon = ImageUtil.getResourceStreamFromClass(TimeTrackingPlugin.class, "pause_icon.png");
+		BufferedImage resetIcon = ImageUtil.getResourceStreamFromClass(TimeTrackingPlugin.class, "reset_icon.png");
+		BufferedImage startIcon = ImageUtil.getResourceStreamFromClass(TimeTrackingPlugin.class, "start_icon.png");
+		BufferedImage addIcon = ImageUtil.getResourceStreamFromClass(TimeTrackingPlugin.class, "add_icon.png");
+
+		DELETE_ICON = new ImageIcon(deleteIcon);
+		DELETE_ICON_HOVER = new ImageIcon(ImageUtil.grayscaleOffset(deleteIcon, -100));
```

Comment:
The behavior of `grayscaleOffset` has changed somewhat due to my PR. Double-check to see if these should still be using this method (which now explicitly reduces RGB values, *and not alpha*) or if they should use `alphaOffset` instead.